script_component.hpp

```diff
@@ -14,8 +14,8 @@
 
 #include "\x\cba\addons\main\script_macros_mission.hpp"
 
-#define PATHTOTACF(var1,var2) PATHTOF_SYS(\x\tac\addons,var1,var2)
-#define QPATHTOTACF(var1,var2) QUOTE(PATHTOTACF(var1,var2))
+#define PATHTOTACR(var1,var2) PATHTOF_SYS(\x\tac\addons,var1,var2)
```

Comment:
Nothing important, just want to say `F` at the end means "file", there is none. (I don't care if there is, depends if you do.)

Reply:
I was wondering what that was for, will update it.